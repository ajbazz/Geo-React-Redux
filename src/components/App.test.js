import React from "react";
import { create } from "react-test-renderer";
import Rating from "./listing_card/rating";

describe("Rating component", () => {
  test("it matches the snapshot", () => {
    const component = create(<Rating rating="4.9" ratePercent="98%" />);
    expect(component.toJSON()).toMatchSnapshot();
  });
});