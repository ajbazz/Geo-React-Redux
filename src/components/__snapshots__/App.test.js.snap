// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Feature component it matches the snapshot 1`] = `
<div
  className="sc-dnqmqq hUMssh"
>
  <div
    className="sc-gZMcBi sc-gqjmRU eZHFaU"
  >
    <span>
      ★
    </span>
    <span>
      ★
    </span>
    <span>
      ★
    </span>
    <span>
      ★
    </span>
    <span>
      ★
    </span>
  </div>
  <div
    className="sc-gZMcBi jKXrHx"
  >
    <span>
      ★
    </span>
    <span>
      ★
    </span>
    <span>
      ★
    </span>
    <span>
      ★
    </span>
    <span>
      ★
    </span>
    <span
      className="sc-VigVT gKVEUn"
    >
      0
    </span>
  </div>
</div>
`;

exports[`Rating component it matches the snapshot 1`] = `
<div
  className="sc-dnqmqq hUMssh"
>
  <div
    className="sc-gZMcBi sc-gqjmRU eZHFaU"
  >
    <span>
      ★
    </span>
    <span>
      ★
    </span>
    <span>
      ★
    </span>
    <span>
      ★
    </span>
    <span>
      ★
    </span>
  </div>
  <div
    className="sc-gZMcBi jKXrHx"
  >
    <span>
      ★
    </span>
    <span>
      ★
    </span>
    <span>
      ★
    </span>
    <span>
      ★
    </span>
    <span>
      ★
    </span>
    <span
      className="sc-VigVT gKVEUn"
    >
      0
    </span>
  </div>
</div>
`;
